bin/publish.ml

```diff
@@ -157,8 +156,9 @@ let man =
     `P "$(mname) $(tname) [$(i,OPTION)]... [$(i,ARTEFACT)]...";
     `S Manpage.s_description;
     `P
-      "The $(tname) command publishes package distribution archives and other \
-       artefacts.";
+      ( Deprecate_delegates.artefacts_warning
```

Comment:
A separate paragraph might be better here!

Reply:
Hmm ok maybe not but what would make sense it to move it after the text below so that we have at least mentioned the alternative artifacts!

Reply:
Yes, that's better. I've moved it below and I've also moved all warnings behind the concept they are for.